src/libs/actions/IOU.js

```diff
@@ -802,6 +913,17 @@ function getSendMoneyParams(report, amount, currency, comment, paymentMethodType
             key: optimisticChatReportData.key,
             value: {pendingFields: null},
         });
+        failureData.push({
+            onyxMethod: CONST.ONYX.METHOD.MERGE,
+            key: optimisticChatReportData.key,
+            value: {
+                errorFields: {
+                    createChat: {
+                        [DateUtils.getMicroseconds()]: Localize.translateLocal('report.genericCreateReportFailureMessage'),
+                    },
+                },
+            },
+        });
```

Comment:
This caused a regression https://github.com/Expensify/App/issues/18839. We already set an error for the report action (in line 883) and we are setting another error for the report itself. This resulted in redundant error messages.